src/dune_lang/format.ml

```diff
@@ -1,30 +1,28 @@
 open! Stdune
-open! Import
 open Pp.O
+module Format = Stdlib.Format
 
 type dune_file =
   | OCaml_syntax of Loc.t
-  | Sexps of Dune_lang.Cst.t list
+  | Sexps of Cst.t list
 
 let parse_lexbuf lb =
   if Dune_lexer.is_script lb then OCaml_syntax (Loc.of_lexbuf lb)
-  else Sexps (Dune_lang.Parser.parse lb ~mode:Cst)
+  else Sexps (Parser.parse lb ~mode:Cst)
 
 let parse_file path_opt =
   match path_opt with
-  | Some path -> Io.Untracked.with_lexbuf_from_file path ~f:parse_lexbuf
```

Comment:
What happened here? Can we keep the "untracked" bits? I find them useful when reading code.

Reply:
It's because the `Untracked` stuff lives in Dune_engine and it's impossible to use it here without introducing circular dependencies.

Reply:
I see. We could similarly shadow untracked file operations here but perhaps it's less of an issue outside of the build engine.

Reply:
What I'm doing is inlining some of these IO functions to the callers. It makes the code more clear and removes the surface area for mistakes. The only remaining function in this module that will do IO will have the name `format_action` and I think we already know that actions are allowed untracked IO.